hibernate-core/src/test/java/org/hibernate/test/inheritance/DenormalizedTablePhysicalIncludedTableConstraintTest.java

```diff
@@ -26,10 +22,6 @@
  * @author Nathan Xu
  */
 @TestForIssue( jiraKey = "HHH-14234" )
-@SkipForDialects( {
-		@SkipForDialect( value = MySQLDialect.class, comment = "skip it for it support constraint name uniqueness in table, not db" ),
-		@SkipForDialect( value = MariaDBDialect.class, comment = "skip it for it support constraint name uniqueness in table, not db" )
-} )
```

Comment:
Seems the above two dialects are not the only ones which don't treat uniqueness constraint name unique within db. Anyway, including them doesn't hurt.